src/app/shared/shared.module.ts

```diff
@@ -26,7 +28,8 @@ import { TabComponent } from './tab';
     ImageUploadComponent,
     SpinnerComponent,
     TabComponent,
-    TimeAgoPipe
+    TimeAgoPipe,
+    HoverDirective
```

Comment:
This is probably OCD Ryan talking ... but i was aiming for keeping the imports/exports sorted alphabetically in these module files.